test/truefi2/LoanFactory2.test.ts

```diff
@@ -99,21 +100,29 @@ describe('LoanFactory2', () => {
     })
 
     it('prevents 0 loans', async () => {
-      await expect(loanFactory.connect(borrower).createLoanToken(pool.address, 0, 100))
+      await expect(loanFactory.connect(borrower).createLoanToken(pool.address, 0, 100, MAX_APY))
         .to.be.revertedWith('LoanFactory: Loans of amount 0, will not be approved')
     })
 
     it('prevents 0 time loans', async () => {
-      await expect(loanFactory.connect(borrower).createLoanToken(pool.address, parseEth(123), 0))
+      await expect(loanFactory.connect(borrower).createLoanToken(pool.address, parseEth(123), 0, MAX_APY))
         .to.be.revertedWith('LoanFactory: Loans cannot have instantaneous term of repay')
     })
 
     it('prevents fake pool loans', async () => {
       const fakePool = await new TrueFiPool2__factory(owner).deploy()
-      await expect(loanFactory.connect(borrower).createLoanToken(fakePool.address, parseEth(123), 0))
+      await expect(loanFactory.connect(borrower).createLoanToken(fakePool.address, parseEth(123), 0, MAX_APY))
         .to.be.revertedWith('LoanFactory: Loans cannot have instantaneous term of repay')
     })
```

Comment:
Since we are at it, can you fix this test? You need to change time to something greater than 0, than the revert message will change to appropriate one.

Reply:
Done.